lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -104,9 +129,10 @@ describe('+ copySync()', () => {
           assert(!fs.existsSync(path.join(destSub, j.toString())))
         }
       }
+      done()
```

Comment:
Revert this change. Using `done` isn't needed since we are using sync methods here.